projects.json

```diff
@@ -941,7 +941,13 @@
       {
         "action": "BuildSwiftPackage",
         "configuration": "release",
-        "tags": "sourcekit-disabled swiftpm"
+        "tags": "sourcekit-disabled swiftpm",
+        "xfail": [
+          {
+            "issue": "rdar://81180448",
+            "compatibility": ["5.2", "5.3"],
+            "branch": ["main", "release/5.5"]
+          }
```

Comment:
Missing `]` for xfails.

Reply:
looks like I did the copy-and-paste wrong.